gqcpy/src/Basis/Transformations/UTransformation_bindings.cpp

```diff
@@ -41,9 +41,23 @@ void bindUTransformation(py::module& module) {
     // Define the Python class for `UTransformation`.
     py::class_<UTransformation<double>> py_UTransformation_d {module, "UTransformation_d", "A type that encapsulates transformation matrices for the alpha- and beta-parts of spin-orbital bases."};
 
+    py_UTransformation_d
+        .def("inverse",
+             &UTransformation<double>::inverse,
+             "Return the inverse transformation of this transformation matrix.")
+
+        .def(
+            "isUnitary",
+            [](const UTransformation<double>& T, const double threshold = 1.0e-12) {
+                return T.isUnitary(threshold);
+            },
+            "Return if this transformation is unitary, within the given threshold");
```

Comment:
This is a copy-paste from the implementation in `bindBasisTransformableInterface`. Could you wrap these methods into a re-usable function and place it into `interfaces.hpp`?

Reply:
It was a copy-paste from `SimpleTransformationInterface`, but since `UTransformation` is a `SpinResolved`-like object, I could not use this interface. I've made a new interface consisting of bindings for SpinResolved-like transformations like `UTransformation` is.

Reply:
I think I didn't express myself clearly enough.

In your current implementation, there are still 2 occurrences of a `"isUnitary"` and an `"isInverse"` binding. Could you create a function, for example `bindBasisTransformableOperationsInterface`, collecting these two bindings? You can then use this function inside the implementation of `bindSimpleTransformationInterface` and your new binding.

If this doesn't make sense, let me know and we'll arrange a short meeting to discuss this.

Reply:
This is indeed a better way, and is now incorporated in the last commit.